### DISCUSSION TITLE
Feature request: Accept negative values for `outline-offset`

### DISCUSSION THREAD

**Discussion by wongjn** [tailwindlabs/tailwindcss#9128] [Ideas]
[`outline-offset`](https://developer.mozilla.org/en-US/docs/Web/CSS/outline-offset) supports a negative value but one cannot use negative prefix like one can do with margins, for example:

`m-2` ✅
`outline-offset-2` ✅
`-m-2` ✅
`-outline-offset-2` ❌

If I see it correctly, it would involve adding `supportsNegativeValues: true` into here:
https://github.com/tailwindlabs/tailwindcss/blob/7924e7b40c72236b4e3d1aec707d59d197782fe5/src/corePlugins.js#L1991-L1993

**Comment by adamwathan** [comment]
Sounds good to me, welcome a PR for it 👍
